db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -8148,82 +8148,15 @@ private long getBootParameter(Properties startParams,String key,boolean required
-     * In that case, the timestamp will remain the same, we will just increment n at the end of the name.
-     * <p/>
-     * Following is how we get around the problem of more than 10 system generated name requestes at the same time:
-     * When the database boots up, we start a counter with value -1 for the last digit in the generated name.
-     * We also keep the time in millisec to keep track of when the last system name was generated. At the
-     * boot time, it will be default to 0L. In addition, we have a calendar object for the time in millisec
-     * That calendar object is used to fetch yy, mm, dd, etc for the string SQLyymmddhhmmssxxn
-     * <p/>
-     * When the first request for the system generated name comes, time of last system generated name will be less than
-     * the current time. We initialize the counter to 0, set the time of last system generated name to the
-     * current time truncated off to lower 10ms time. The first name request is the only time we know for sure the
-     * time of last system generated name will be less than the current time. After this first request, the next request
-     * could be at any time. We go through the following algorithm for every generated name request.
-     * <p/>
-     * First check if the current time(truncated off to lower 10ms) is greater than the timestamp for last system generated name
-     * <p/>
-     * If yes, then we change the timestamp for system generated name to the current timestamp and reset the counter to 0
-     * and generate the name using the current timestamp and 0 as the number at the end of the generated name.
-     * <p/>
-     * If no, then it means this request for generated name has come at the same time as last one.
-     * Or it may come at a time less than the last generated name request. This could be because of seasonal time change
-     * or somebody manually changing the time on the computer. In any case,
-     * if the counter is less than 10(meaning this is not yet our 11th request for generated name at a given time),
-     * we use that in the generated name. But if the counter has reached 10(which means, this is the 11th name request
-     * at the same time), then we increment the system generated name timestamp by 10ms and reset the counter to 0
-     * (notice, at this point, the timestamp for system generated names is not in sync with the real current time, but we
-     * need to have this mechanism to get around the problem of more than 10 generated name requests at a same physical time).
+     * Returns a unique system generated name of the form SQL[UUID]. The uniqueness must be guaranteed across all region servers.
+     * com.splicemachine.db.iapi.services.uuid.UUIDFactory is used to generate an unique ID.
      *
      * @return system generated unique name
      */
     @Override
     public String getSystemSQLName(){
-        StringBuilder generatedSystemSQLName=new StringBuilder("SQL");
-        synchronized(this){
```

Comment:
The generated SQL name needs to be upper case. If possible, use numbers following "SQL". One example to generate a random UUID
java.util.UUID.randomUUID().toString().replaceAll("-","")

The case sensitivity problem caused some IT failures

Reply:
This has been fixed. No failed ITs now: TEST SUCCEEDED +1